### ISSUE TITLE
Werkzeug 3.0.1 breaking change results in the application refusing to run due to unrecognised import.

### ISSUE THREAD

**Issue by josh-allan** [clegaspi/saml_reader#85]
If building the Saml Reader from scratch, the dependencies will install `Werkzeug` version 3.0.1 as the packages will just update to the most recent version. This introduces breaking changes in the import of packages specifically in the import for `Werkzeug.urls` functions.  

When running `saml_reader` with version 3.0.1 it throws a stack trace that results in:

```
ImportError: cannot import name 'url_quote' from 'werkzeug.urls' (/opt/homebrew/lib/python3.9/site-packages/werkzeug/urls.py)
```

This can be easily resolved by downgrading the package back to 2.2.2. I have opened a PR to resolve this.

**Comment by clegaspi** [comment]
I think that by upgrading dash and dash-dependencies, this fixes this, as I did in #88 but I will look at it more closely on an M1 machine too.

**Comment by clegaspi** [comment]
Confirmed this as working on M1. I will revert the documentation changes shortly.
